CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing
+
+Thanks for wanting to contribute to Vac specs!
+
+## Issues and PRs
+
+Open an issue with a problem statement and suggested approach, where relevant, and a PR that links to this issue.
+
+If it is a minor cosmetic change this can be pushed as a PR directly.
+
+To make sure people don't miss your PR, ask for review from people of the Vac team in the PR.
+
+## Spec lifecycle
+
+Every spec has its own lifecycle that shows its maturity. We indicate this in a similar fashion to [COSS Lifecycle](https://rfc.unprotocols.org/spec:2/COSS/):
```

Comment:
The COSS Lifecycle link has been broken for me for some time. It may be temporary, but I refer to the docs on GitHub directly [here](https://github.com/unprotocols/rfc/blob/master/2/README.md)

Reply:
yeah, doesn't work for me either